pages/linux/pihole.md

```diff
@@ -1,15 +1,15 @@
 # pihole
 
 > Terminal interface for the Pi-hole ad-blocking DNS server.
-> More information: <https://pi-hole.net>.
+> More information: <https://docs.pi-hole.net/core/pihole-command/>.
 
 - Check the Pi-hole daemon's status:
 
 `pihole status`
 
-- Update Pi-hole:
+- Update Pi-hole (software and adlists):
 
-`pihole updatePihole`
+`pihole -up`
```

Comment:
```suggestion
`pihole -up`
```

I was the one to add this example to this page, and even I agree that you should update like this, the Pi-Hole arguments are weirdly named but using `-up` is better.

Reply:
Yeah, I had never even heard of `updatePihole` before I read the tldr.